src/fu-util-common.c

```diff
@@ -1252,6 +1252,12 @@ fu_util_device_to_string (FwupdDevice *dev, guint idt)
 		}
 	}
 
+	tmp = fwupd_device_get_physical_id (dev);
+	if (tmp != NULL) {
+		/* TRANSLATORS: Physical ID for hardware */
+		fu_common_string_append_kv (str, idt + 1, _("Physical ID"), tmp);
```

Comment:
Along the lines of my other comment if we are showing this to users, I want to make sure that we show something useful.  So depending on how this is looking for most devices, we might want to have a decoder function to walk sysfs and look for `name` attributes or to use udev to build some nice strings to present.

Reply:
I think we need to nail down exactly what the physical ID **is** before we even think about displaying it.